Description: fix sys.path for tests
 Fix sys.path in the test support module so that is matches paths that are in
 the Debian package.
Author: Jakub Wilk <jwilk@debian.org>

--- a/test/DocutilsTestSupport.py
+++ b/test/DocutilsTestSupport.py
@@ -52,12 +52,9 @@
 os.chdir(testroot)
 if sys.version_info >= (3,0):
     sys.path.insert(0, os.path.normpath(os.path.join(testroot,
-                                                     '..', 'build', 'lib')))
-    sys.path.append(os.path.normpath(os.path.join(testroot, '..',
-                                                  'build', 'lib', 'extras')))
+                                                     '..', 'build', 'py3')))
 else:
-    sys.path.insert(0, os.path.normpath(os.path.join(testroot, '..')))
-    sys.path.append(os.path.normpath(os.path.join(testroot, '..', 'extras')))
+    sys.path.insert(0, os.path.normpath(os.path.join(testroot, '..', 'build', 'py2')))
 sys.path.insert(0, testroot)
 
 try:
